src/Android/Xamarin.Android/Samples/Data/StatisticalQuery/README.md

```diff
@@ -0,0 +1,12 @@
+#Statistical query
```

Comment:
We'll want to update the formatting on the readmes. Primarily we need to:

* add space between # and title text
* remove the tags at the bottom

This can either be done manually, or you can re-run the latest version of the readme generator.